pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -310,6 +310,22 @@ func toBeDisabled() toolchainv1alpha1.Condition {
 	}
 }
 
+func toBeProvisionedNotificationCreated() toolchainv1alpha1.Condition {
```

Comment:
@alexeykazakov not sure if this name is good enough. not a huge fan but i couldn't think of anything better any suggestions?

Reply:
Yeah... That "toBe" prefix IMO is a bit confusing. But let's keep it for the consistency sake.

Reply:
Yeah agreed, sounds good.